addons/account_peppol/wizard/account_move_send.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64encode
+
+from odoo import api, fields, models, _
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.exceptions import UserError
+
+
+class AccountMoveSend(models.Model):
+    _inherit = 'account.move.send'
+
+    checkbox_send_peppol = fields.Boolean(
+        string='Send via PEPPOL',
+        help='Send the invoice via PEPPOL',
+    )
+    peppol_proxy_state = fields.Selection(related='company_id.account_peppol_proxy_state')
+    peppol_warning = fields.Char(
+        string="Warning",
+        compute="_compute_peppol_warning",
+    ) # technical field needed for computing a warning text about the peppol configuration
+
+    # -------------------------------------------------------------------------
+    # COMPUTE METHODS
+    # -------------------------------------------------------------------------
+
+    @api.depends('checkbox_send_peppol')
+    def _compute_checkbox_ubl_cii_xml(self):
+        # Extends account.move.send wizard - auto check checkbox_ubl_cii_xml
+        # if checkbox_send_peppol is selected
+        super()._compute_checkbox_ubl_cii_xml()
+        for wizard in self:
+            if wizard.checkbox_send_peppol and not wizard.checkbox_ubl_cii_xml:
```

Comment:
why `not wizard.checkbox_ubl_cii_xml`
To me, you should send using peppol only if `checkbox_ubl_cii_xml` is checked

Reply:
That's what the code is doing, it is checking it if it wasn't checked